### ISSUE TITLE
Not an issue comment

### ISSUE THREAD

**Issue by Orion99282i** [tpd0618/touhou-pixel-dungeon#4]
I hope this mod will be discovered throughout the touhou community..And i think this has potential to become popular.
Me and 5 other dudes are waiting for the rework 😎 this **December** Thank you for creating the project and updating it till this day

**Comment by Gamby380** [comment]
I agree, i made this account just to comment here. Hope they didn't mean the last update by "final", the new spritework still seems to be work in progress as not every character got their sprite replaced so it seems like there'll be more updates and it gives me hope. Old sprites are still really charming, i love their goofy looks.
